azure-cli/synapse/Publish/deploy.synapse.yml

```diff
@@ -0,0 +1,39 @@
+trigger:
+  branches:
+    include:
+    - workspace_publish # Deployment of Synapse ETL processes uses ARM templates from workspace_publish branch
+
+resources:
+  repositories:
+  - repository: PublishBranch
+    type: git
+    name: 'Azure-Project-Name/Repo-name'
+    ref: workspace_publish
+
+
+name: 1.0$(Rev:.r) 
+
+stages:
+  - stage: Staging
+    # Ignore PR flow
+    condition: "and(succeeded(), ne(variables['Build.Reason'], 'PullRequest'))"
+    jobs:
+
+    - template: pipelines/templates/deploy.synapse.template.yml
+      parameters:
+        environmentName: staging
+        environmentType: DevTest
+        subscription: ''
```

Comment:
Also "# Change this" here? Maybe with a dummy example - same goes for Production section